hkmc2/shared/src/main/scala/hkmc2/semantics/ucs/DesugaringBase.scala

```diff
@@ -22,54 +22,50 @@ trait DesugaringBase(using state: State):
   protected final def app(l: Term, r: Term, label: Str): Term.App = app(l, r, FlowSymbol(label))
   protected final def app(l: Term, r: Term, s: FlowSymbol): Term.App = Term.App(l, r)(App(Empty(), Empty()), s)
 
-  /** Make a term looks like `globalThis.Predef.MatchResult` with its symbol. */
+  /** Make a term looks like `runtime.MatchResult` with its symbol. */
```

Comment:
I think all these comments are missing a "that"... (incl. the other comments below)
```suggestion
  /** Make a term that looks like `runtime.MatchResult` with its symbol. */
```

Reply:
This is not resolved by your commit. Read my message again.

Reply:
Oh, I forgot to push my commits yesterday.